pymongoexplain/commands.py

```diff
@@ -51,8 +53,13 @@ def __init__(self, collection: Collection, filter, update,
             value = kwargs[key]
             if key == "bypass_document_validation":
                 return_document[key] = value
+            elif key == "hint":
+                return_document["updates"][0]["hint"] = value if \
+                    isinstance(value, str) else SON(value)
```

Comment:
This does not have the same behavior as Pymongo. Check out how the Collection class handles the "hint" option.

Reply:
I am confused about this comment. This matches the output produced by Pymongo.

Reply:
Check out _index_document and where that method is used: https://github.com/mongodb/mongo-python-driver/blob/956ce3d4b0edfa9c1d946109db743f82ed0bfc0a/pymongo/helpers.py#L79